.github/proverCiScripts/benchmarks_result_reporting/reporting_modules.py

```diff
@@ -0,0 +1,212 @@
+from sqlalchemy import create_engine, text
+import warnings, time, os
+
+
+warnings.simplefilter(action='ignore', category=FutureWarning)
+warnings.simplefilter(action='ignore', category=UserWarning)
+pd.options.mode.chained_assignment = None
+
+def pgsql_engine(pgsqldb):
+    '''
+    creates a psql engine instance
+    '''
+    user     = pgsqldb['user']
+    password = pgsqldb['password']
+    host     = pgsqldb['host']
+    database = pgsqldb['database']
+    engine   = create_engine(f'postgresql://{user}:{password}@{host}:5432/{database}')
+
+    return engine
+
+def prepare_result_dataframe(test_result, sysstat,env, test_id):
+    '''
+    prepares postgres data (in the form of dataframe) for table circuit_benchmarks
+    '''
+    try:
+        if test_result['result'] == 'PASSED':
+            r = {
+                'pull_request'          : test_result['pull_request'],
+                'test_id'               : test_id,
+                'circuit'               : test_result['circuit'],
+                'degree'                : test_result['degree'],
+                'test_result'           : test_result['result'],
+                'test_date'             : datetime.datetime.now().date(),
+                'setup_gen'             : test_result['setup_gen'],
+                'proof_gen'             : test_result['proof_gen'],
+                'proof_ver'             : test_result['proof_ver'],
+                'max_ram'               : sysstat['max_ram'],
+                'cpu_all_Average'       : sysstat['cpu_all_Average'],
+                'cpu_all_Max'           : sysstat['cpu_all_Max'],
+                'cpu_count'             : sysstat['cpu_count'],
+                'sysstats_url'          : f'{env["grafana_dashboard_prefix"]}{test_id}',
+                'logsurl'               : f'{env["s3endpoint"]}{test_id}.tar.gz'
+            }
+        else:
+            r = {
+                'pull_request'          : test_result['pull_request'],
+                'test_id'               : test_id,
+                'circuit'               : test_result['circuit'],
+                'degree'                : test_result['degree'],
+                'test_result'           : test_result['result'],
+                'test_date'             : datetime.datetime.now().date(),
+                'setup_gen'             : test_result['setup_gen'],
+                'proof_gen'             : test_result['proof_gen'],
+                'proof_ver'             : test_result['proof_ver'],
+                'max_ram'               : sysstat['max_ram'],
+                'cpu_all_Average'       : sysstat['cpu_all_Average'],
+                'cpu_all_Max'           : sysstat['cpu_all_Max'],
+                'cpu_count'             : sysstat['cpu_count'],
+                'sysstats_url'          : f'{env["grafana_dashboard_prefix"]}{test_id}',
+                'logsurl'               : f'{env["s3endpoint"]}{test_id}.tar.gz'
+            }
```

Comment:
Both `r`s look the same in both branches.

Reply:
Thanks @ChihChengLiang , the conditional was obviously redundant. Has been removed with[14c169c](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1166/commits/14c169cd5dad5e411430ec1209eb970a7ec1ba7d)
which also fixes a typo and allows parser to read the full prover log